app/controllers/authenticate_controller.rb

```diff
@@ -223,6 +250,46 @@ def handle_authentication_error(err)
     end
   end
 
+  def audit_authn_error(err)
+    log_audit_event(
+      ::Audit::Event::Authn::Authenticate.new(
+        authenticator_name: authenticator_input.authenticator_name,
+        service: authenticator_input.webservice,
+        role_id: audit_role_id,
+        client_ip: authenticator_input.client_ip,
+        success: false,
+        error_message: err.message
+      )
+    )
+  end
```

Comment:
can we have an alignment between this and the update_config audits? there you have a method called `log_audit_event` and you give it an event as input.  we can do the same here.